.gitignore

```diff
@@ -129,4 +129,4 @@ dmypy.json
 .idea/
 
 # JUnit
-junit.xml
+junit.xml
```

Comment:
There is again a change with final newline. You can use a plugin for your code editor with [EditorConfig](editorconfig.org) support, there are plenty of them on the site.

Reply:
Sorry, I didn't know about this, maybe it would be a good idea to add this to the contributing section (or it's obvious from the file extension)